### PR TITLE
Add CPU govenor setup playbook

### PR DESCRIPTION
**Pull request by cloudnull**
The CPU govenor setup was being done by request post installation. This
change simply makes this operational tuning part of our standard
deployment.

Signed-off-by: Kevin Carter <kevin.carter@rackspace.com>

### CODE REVIEW

**Comment by cloudnull** [comment]
recheck_bionic_no_artifacts_swift_deploy

File: scripts/deploy-rpco.sh [hunk 1]
```diff
@@ -46,6 +46,12 @@ pushd "${SCRIPT_PATH}/../playbooks"
   else
     openstack-ansible site-openstack.yml
   fi
+
+  # Deploy RPC operational modifications
+  openstack-ansible site-ops.yml
+
+  # Deploy RPC Logging
+  openstack-ansible site-logging.yml
```
Comment (prometheanfire) [inline scripts/deploy-rpco.sh]: this seems unrelated (site-logging)
